packages/components/layout/Section.ts

```diff
@@ -18,19 +18,23 @@ export class Section extends Component {
 		return css`
 			:host {
 				display: block;
+				position: relative;
 			}
 
-			slot[name=heading] {
-				display: block;
-				flex: 1;
+			mo-heading {
+				margin: 8px 4px 8px 0;
```

Comment:
1. Margin nach oben ist in dem Fall ein Bad-Practice, da die Komponente zwingt nach außen Margin zu haben, was nicht die Verantwortung dieser Komponente ist. Die Abstände zwischen Komponenten sind von außen durch Layout-Komponenten z.B. Flex und Grid zu regeln.

2. Margin nach unten wurde hier missbraucht um entweder den Abstand zwischen der Headline und dem Content zu regeln, stattdessen bitte den `gap` vom außeren Flex benutzen. ODER um die Zeilenhöhe zu vergrößert, stattdessen bitte `line-height` benutzen.

3. Margin nach rechts wurde hier missbraucht um den Abstand zwischen 2 slots zu regeln, hierzu bitte den `gap` von Flex nutzen.

Reply:
Ich habe den Abstand eingebaut, aber irgendwie zieht er nicht zwischen den Slots